src/topology/homeomorph.lean

```diff
@@ -230,6 +231,14 @@ by rw [← preimage_symm, preimage_interior]
 lemma preimage_frontier (h : α ≃ₜ β) (s : set β) : h ⁻¹' (frontier s) = frontier (h ⁻¹' s) :=
 h.is_open_map.preimage_frontier_eq_frontier_preimage h.continuous _
 
+@[to_additive]
+lemma _root_.has_compact_mul_support.comp_homeomorph {M} [has_one M] {f : β → M}
```

Comment:
this should be true more generally when phi is a closed embedding. Could you prove the more general version, and deduce this one afterwards?